### ISSUE TITLE
Cured Villagers are Invisible

### ISSUE THREAD

**Issue by SpectresGaming** [WildBamaBoy/minecraft-comes-alive#937]
Going through the process of curing a villager seems to be broken, as doing such for me has yielded an invisible farmer type male named 'Steve'. There is no way to customize the villager as opening the window and changing even one setting causes the game to CTD. There is no other problem, and this has happened to me three times in two different worlds.

Forge Version 14.23.1.2556 (tested with 2491)
img: https://i.imgur.com/PZVwTyw.png

**Comment by SpectresGaming** [comment]
I want to also note that this only happens with base game Zombie Villagers, and that attacking this invisible 'Steve the Farmer' always yields a strange red hue added to the game's view filter, almost like an effect, though nothing can be done to reset it except for a full game restart.

**Comment by Whiteshadow64** [comment]
I too am having this problem, but on Forge 14.23.1.2587 and 14.23.1.2589. I did a few tests in game and found that wild zombie villagers and originally villagers turned zombie both turn into Steve the invisible farmer. Additionally, the red tint activates whenever Steve takes any damage, and goes away if any other mob takes damage.

Were you using any mods other than MCA? If so, could you list them so we could compare the mods we used?

**Comment by Whiteshadow64** [comment]
As an update to my previous post, I tried running MCA 1.12x 5.3.1 with no mods other than RadixCore-1.12x 2.2.1 to see what would happen. Interestingly, the cured villager would still be named Steve the Farmer, instead of being invisible, he had the standard purple and black missing texture. Additionally, dealing damage to him did not tint the screen red.

Edit: I have tried the other versions of MCA and Radix for 1.12. Still the same issue

**Comment by NotAVerb** [comment]
I also have this issue. [1.12.2 MCA 5.3.1] Cured a zombie, got an invisible farmer. I don't think he was named Steve though. -edit: He was in fact named Steve. I took a screenshot but forgot about it after encountering the error log. -
I also got this -massive- error log.
https://gist.github.com/NotAVerb/45f4607e6924c85145f1f7b35f5cbd24
While I was playing there was an infinite error loop until I exited the game.
